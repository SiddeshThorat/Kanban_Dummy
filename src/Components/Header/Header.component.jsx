import React from 'react';
import './Header.styles.css';

const Header = ({children}) => (
    <div className="headerContainer">
        {children}
    </div>
)

export default Header;