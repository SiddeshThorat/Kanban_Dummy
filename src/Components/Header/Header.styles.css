.headerContainer{
    background-color:  #a19ea1;
    height: 50px;
    border-radius:20px 20px 0 0;
    font-weight: bold;
    font-size: 20px;
    font-family: serif;
    vertical-align: middle;
    text-align: center;
    display: table-cell;
    width: 400px;
}