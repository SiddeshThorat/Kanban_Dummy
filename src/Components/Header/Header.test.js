import React from 'react';
import { shallow } from 'enzyme'
import Header from'./Header.component';

describe("Testing Header component",() => {
    it("Snapshot testing of Header component",() => {
        expect(shallow(<Header />)).toMatchSnapshot();
    })
})


