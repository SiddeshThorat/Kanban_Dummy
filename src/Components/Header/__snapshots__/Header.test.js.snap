// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component Snapshot testing of Header component 1`] = `
<div
  className="headerContainer"
/>
`;
